### ISSUE TITLE
DropOut layer and others support

### ISSUE THREAD

**Issue by ToGoOrNotToGo** [Cr33zz/Neuro#1]
This is a very interesting project!
Are you implementing more layers into this project, like into Neuro_?
E.g. DropOut layer?

**Comment by Cr33zz** [comment]
Hey, unfortunately, I stopped developing this project after porting it to C++. I had to focus on a single project given the limited time I have.

**Comment by ToGoOrNotToGo** [comment]
That is understandable. However, there are already some deep learning libraries in C++ and Python, but not that many in C#. Isn't it?
But your source code is very clean in both languages and should be relatively easy to convert if necessary (maybe even with tools like "C ++ to C # Converter" from Tangible)...

**Comment by ToGoOrNotToGo** [comment]
OK it is not that simple to convert from C++ to C#. So, please can you add a DropOut layer to the C# project too?

**Comment by Cr33zz** [comment]
Yeah, I suppose I could do that; however, I will need some time (a few weeks) as right now, I'm away with no access to development PC. You could also take a look at non-computational-graph branch of Neuro_ and dropout layer implementation there. Should be easier to port it than the current implementation from master.
